process/sync/metablock.go

```diff
@@ -195,11 +194,21 @@ func (boot *MetaBootstrap) SyncBlock(ctx context.Context) error {
 }
 
 func (boot *MetaBootstrap) syncAccountsDBs(key []byte, id string) error {
+	userAccountsStorerIdentifier, err := boot.getStorerIdentifier(dataRetriever.UserAccountsUnit)
+	if err != nil {
+		return err
+	}
+
+	peerAccountsStorerIdentifier, err := boot.getStorerIdentifier(dataRetriever.PeerAccountsUnit)
+	if err != nil {
+		return err
+	}
+
 	// TODO: refactor this in order to avoid treatment based on identifier
 	switch id {
```

Comment:
this still won't work if I decide, for example, to change the name of the AccountsTrie DB in the config.toml